setup.py

```diff
@@ -1,31 +1,44 @@
 from setuptools import setup, find_packages
-import os.path
-
-def read(path):
-    return open(
-        os.path.join(os.path.dirname(__file__), *path.split('/'))).read()
-
 
 setup(
     name="zc.set",
-    version="0.1dev",
+    version="0.2",
     license='ZPL 2.1',
     long_description='\n\n'.join([
-        read('CHANGES.txt'),
-        read('src/zc/set/README.txt'),
+        open('CHANGES.txt').read(),
+        open('src/zc/set/README.txt').read(),
         ]),
```

Comment:
It might be a good idea to add `url='https://github.com/zopefoundation/zc.set',` here.